client/src/ui/messaging/MessageCard.tsx

```diff
@@ -13,6 +13,13 @@ export interface MessageCardProps {
   scrollDown?: () => void;
 }
 
+export interface Message {
```

Comment:
This type is redundant since the type for the MessageCard is the MessageCardProps interface. If you need to get the types from message, import the Database object from "@/lib/types/database.types" from the file you need it